settings.gradle.kts

```diff
@@ -31,17 +32,25 @@ dependencyResolutionManagement {
 
     // Catalogs
     versionCatalogs {
+        create("libs") {
+            library("logstash-logback-encoder", "net.logstash.logback", "logstash-logback-encoder").version("7.4")
+        }
         create("testLibs") {
-            val kotestApiVersion = "5.6.2"
+            val kotestApiVersion = "5.8.0"
 
             library("kotest-framework-api", "io.kotest", "kotest-framework-api-jvm").version(kotestApiVersion)
+            library("kotest-assertions-core", "io.kotest", "kotest-assertions-core").version(kotestApiVersion)
 
             library(
                 "kotest-extensions-testcontainers",
                 "io.kotest.extensions",
                 "kotest-extensions-testcontainers"
             ).version("1.3.4")
 
+            // Logging
+            library("microutils-logging", "io.github.microutils", "kotlin-logging-jvm").version("3.0.5")
+
+            library("micronaut-jackson-databind", "io.micronaut", "micronaut-jackson-databind").version("4.4.6")
```

Comment:
micronaut libs are in the mn catalog afaik and do not need to be created here

Reply:
true for micronaut-jackson-databind, microutils-logging is not in micronaut.